it-and-security/lib/macos/scripts/macos-uninstall-fleetd.sh

```diff
@@ -0,0 +1,60 @@
+# Please don't delete. This script is referenced in the guide here: https://fleetdm.com/guides/how-to-uninstall-fleetd
+
+#!/bin/sh
```

Comment:
@allenhouchins do you know what happens if I add a comment over the shebang? Will this "just work"?